projects/valtimo/components/assets/css/carbon.scss

```diff
@@ -14,19 +14,20 @@
  * limitations under the License.
  */
 
-$valtimo-blue-100: #002547;
-$valtimo-blue-90: #002c54;
-$valtimo-blue-80: #003361;
-$valtimo-blue-70: #286198;
-$valtimo-blue-60: #2b79bd;
-$valtimo-blue-50: #61aedf;
-$valtimo-blue-40: #8acff2;
-$valtimo-blue-30: #aadcf6;
-$valtimo-blue-20: #c9e9f9;
-$valtimo-blue-10: #e9f6fd;
+$valtimo-color-100: #002547;
+$valtimo-color-90: #002c54;
+$valtimo-color-80: #003361;
+$valtimo-color-70: #286198;
+$valtimo-color-60: #2b79bd;
+$valtimo-color-50: #61aedf;
+$valtimo-color-40: #8acff2;
+$valtimo-color-30: #aadcf6;
+$valtimo-color-20: #c9e9f9;
+$valtimo-color-10: #e9f6fd;
 
 @use '@carbon/styles/scss/themes';
 @use '@carbon/colors';
+@use 'projects/valtimo/components/assets/css/color';
```

Comment:
@teodor-ritense is this correct?

Reply:
No, it should be the relative path, @peetritense please fix